src/core.js

```diff
@@ -1892,12 +1892,14 @@ define([
             success: function (data) {
                 if (saveType === 'patch') {
                     if (data.status === 'conflict') {
-                        /* todo: display diff and ask instead overwriting */
-                        // var diffHtml = dmp.diff_prettyHtml(
-                        //     dmp.diff_main(formText, data.xform)
-                        // );
-                        _this._hideConfirmDialog();
-                        _this.showOverwriteWarning(_this.send.bind(_this), formText, data.xform);
+                        if (_.isUndefined(data.xform)) {
+                            // unconditionally overwrite if no xform to compare
+                            _this.send(formText, 'full');
```

Comment:
Stand-alone mode would fail to save (due to `data.xform` being undefined in `showOverwriteWarning`) after first save without this.